src/components/Error/Error.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button, makeStyles, Typography } from '@material-ui/core';
+import React, { FC } from 'react';
+
+interface ErrorProps {
+  navigateBack: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  variant?: 'Error' | '404';
+}
+
+const useStyles = makeStyles(({ spacing }) => ({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+  },
+  errorType: {
+    display: 'flex',
+  },
+  boldText: {
+    fontWeight: 'bold',
+    marginRight: spacing(1),
+  },
+  button: {
+    height: spacing(4),
+    width: spacing(16),
```

Comment:
Let's not change the width and height manually. Let's stick to `Button.size` prop options

Reply:
Sure, changed back to default medium size.  It's a little different than the wide button in the design but I think it's fine
<img width="275" alt="Screen Shot 2020-06-23 at 4 31 46 PM" src="https://user-images.githubusercontent.com/43487134/85477600-7a88bb80-b56f-11ea-826f-3eab0bad3bbd.png">
<img width="236" alt="Screen Shot 2020-06-23 at 4 31 38 PM" src="https://user-images.githubusercontent.com/43487134/85477604-7bb9e880-b56f-11ea-9f77-8d248e1a0bd1.png">

Reply:
Yeah that's fine. Maybe size large would be good since it's the only button on screen?

Reply:
here's large, doesn't look good imo
<img width="1412" alt="Screen Shot 2020-06-24 at 3 18 50 PM" src="https://user-images.githubusercontent.com/43487134/85633472-285a9f80-b62e-11ea-84c3-e7a321be99f2.png">
![Uploading Screen Shot 2020-06-24 at 3.18.42 PM.png…]()

Reply:
Yep agreed